dm/ctl/master/query_status_test.go

```diff
@@ -52,26 +52,26 @@ func (t *testCtlMaster) TestWrapTaskResult(c *check.C) {
 	resp.Result = true
 
 	// Should return error when some error occurs in subtask
-	resp.Workers = []*pb.QueryStatusResponse{
+	resp.Sources = []*pb.QueryStatusResponse{
 		{
 			Result: true,
-			Worker: "172.17.0.2:8262",
+			Source: "172.17.0.2:8262",
```

Comment:
seems that the `source` is the worker's address, is it expected?

Reply:
we can refine it next pr

Reply:
Already refined in https://github.com/pingcap/dm/pull/453/commits/e3770179ab92082c9ec6e3f407c9329c8f11f9d7